scripts/mint.ts

```diff
@@ -19,14 +19,92 @@ async function main() {
     config.account,
     config.minter,
     { mint: {} },
-    executeFee
+    executeFee,
+    "mint to sender",
+    coins("100000000", "ustars")
   );
   const wasmEvent = result.logs[0].events.find((e) => e.type === "wasm");
   console.info(
     "The `wasm` event emitted by the contract execution:",
     wasmEvent
   );
 }
-const args = process.argv.slice(7);
-await main();
+
+async function mintTo(recipient: string) {
+  if (!recipient.startsWith("stars")) {
+    const { data } = Bech32.decode(recipient);
+    const starsAddr = Bech32.encode("stars", data);
+    recipient = starsAddr;
+  }
+  console.log("Minting to: ", recipient);
+
+  const gasPrice = GasPrice.fromString("0stars");
+  const wallet = await DirectSecp256k1HdWallet.fromMnemonic(config.mnemonic, {
+    prefix: "stars",
+  });
+  const client = await SigningCosmWasmClient.connectWithSigner(
+    config.rpcEndpoint,
+    wallet
+  );
+  const executeFee = calculateFee(300_000, gasPrice);
+  const result = await client.execute(
+    config.account,
+    config.minter,
+    { mint_to: { recipient: recipient } },
+    executeFee,
+    "mint to",
+    coins("100000000", "ustars")
```

Comment:
magic number?

Reply:
It's hard coded in the contracts... potentially could add to the config?